CHANGELOG.md

```diff
@@ -6,9 +6,15 @@
 
 ## UNRELEASED
 
+### Added
+- Institution tag management: Add search field to institution list
+
 ## 2.20.0
 
-- Added: Admin institution list: Institutions can now be tagged filtered by categories.
-- Fixed: Several bug fixes
-- Chore: Addons can now be installed by automatically, when listed in the `addons.yml` file.
-- Chore: Several major dependencies have been updated.
+### Added
+- Admin institution list: Institutions can now be tagged filtered by categories
+
+### Fixed
+- Several bug fixes
+- Addons can now be installed automatically, when listed in the `addons.yml` file
+- Several major dependencies have been updated
```

Comment:
Why did You change the format? I am not sure if we want this format here. And if we want to format it like so, we should extract "chore" in a separate block.

Reply:
Ok. Wasn't aware of that bit documentation. Thanks for pointing this out.

maybe then "enhancements"? 
I would discuss it, when we have two or three releases, so we have some content to discuss about